fe/src/main/java/org/apache/doris/load/routineload/RoutineLoadJob.java

```diff
@@ -925,31 +946,37 @@ public void setOrigStmt(String origStmt) {
             }
         }
 
-        List<String> row = Lists.newArrayList();
-        row.add(String.valueOf(id));
-        row.add(name);
-        row.add(TimeUtils.longToTimeString(createTimestamp));
-        row.add(TimeUtils.longToTimeString(endTimestamp));
-        row.add(db == null ? String.valueOf(dbId) : db.getFullName());
-        row.add(tbl == null ? String.valueOf(tableId) : tbl.getName());
-        row.add(getState().name());
-        row.add(dataSourceType.name());
-        row.add(String.valueOf(getSizeOfRoutineLoadTaskInfoList()));
-        row.add(jobPropertiesToJsonString());
-        row.add(dataSourcePropertiesJsonToString());
-        row.add(getStatistic());
-        row.add(getProgress().toJsonString());
-        switch (state) {
-            case PAUSED:
-                row.add(pausedReason);
-                break;
-            case CANCELLED:
-                row.add(cancelReason);
-                break;
-            default:
-                row.add("");
+        readLock();
```

Comment:
The strong consistency need be protected in here?

Reply:
Yes, the errorLogUrls may be modified anytime